compiler/q-implant/README.md

```diff
@@ -0,0 +1,25 @@
+# q-implant
+
+_q-implant_ writes quantization parameters and weights (given as .json and .npy files) to a circle model.
+
+## Example
+
+```sh
+q-implant input.circle qparam.json output.circle
+```
+
+`qparam.json` and `*.npy` files must exist in the same directory. For example,
+
+```json
+{
+  "ifm": {
+    "dtype": "uint8",
+    "scale": "0.npy",
+    "zerop": "1.npy",
+    "quantized_dimension": 0
+  },
+  ...
+}
+```
+
+`0.npy` and `1.npy` must exist in the same directory with `qparam.json`.
```

Comment:
from this explanation, there will be two `.npy` files for one operator(node) from the model. for real models, there could be 10s or 100s for nodes in a model.
which would produces lots of files in the process of `q-implant`.
I don't think this is a good approach for users.

How about using `h5` format and produce single file for a model?

Reply:
> How about using h5 format and produce single file for a model?

There is an issue in using h5 format.

tensors can have `/` in their names, but h5 does not allow `/` in the object name. So we have to replace it into another character, which makes hard to recover the original tensor name.

Reply:
let's discuss this in 11530 as this file is not related with "operator level test"